website/templates/base.mako

```diff
@@ -338,6 +338,7 @@
 
 <%def name="javascript_bottom()">
     ### Javascript loaded at the bottom of the page ###
+    <script src="${wafflejs_url}"></script>
```

Comment:
No need to load JS we're not using. Let's leave this commented out until we have a use case for Waffle.